v3/.eslintrc.cjs

```diff
@@ -138,7 +138,8 @@ module.exports = {
       }
     },
     { // eslint configs
-      files: [".eslintrc*.js"],
+      // *.config.ts should be added after cypress.config.ts lint fixes
```

Comment:
The comment in line 140 should be updated now that more than eslint configs are addressed.

Reply:
👍